Detectors/ITSMFT/MFT/assessment/src/MFTAssessment.cxx

```diff
@@ -12,19 +12,38 @@
 #include "MFTAssessment/MFTAssessment.h"
 #include "Framework/InputSpec.h"
 #include "DetectorsBase/GeometryManager.h"
+#include "MFTBase/GeometryTGeo.h"
 #include <Framework/InputRecord.h>
 #include <TPaveText.h>
 #include <TLegend.h>
 #include <TStyle.h>
 #include <TFile.h>
+#include <numeric>
 
 using namespace o2::mft;
+using o2::itsmft::CompClusterExt;
+o2::itsmft::ChipMappingMFT mMFTMapping;
 
 //__________________________________________________________
 void MFTAssessment::init(bool finalizeAnalysis)
 {
   mFinalizeAnalysis = finalizeAnalysis;
   createHistos();
+  //get geometry
+  o2::base::GeometryManager::loadGeometry("", true);
```

Comment:
Not needed, right?

Reply:
Yes it is needed, because when calling convertToglobalClusters, it needs a loaded geometry